meta-lmp-base/recipes-core/os-release/os-release.bbappend

```diff
@@ -4,7 +4,8 @@ OS_RELEASE_UNQUOTED_FIELDS += "IMAGE_ID IMAGE_VERSION"
 # Default values when not built via our factory CI
 LMP_DEVICE_FACTORY ?= "lmp"
 LMP_DEVICE_REGISTER_TAG ?= "master"
-IMAGE ??= "lmp-factory-image"
+LMP_FACTORY_IMAGE ??= "lmp-factory-image"
+IMAGE ??= "${LMP_FACTORY_IMAGE}"
```

Comment:
Should we drop the use of `IMAGE` in this recipe?

Reply:
Yup, should be fine, I got the wrong impression that it was actually IMAGE_ID.

Will drop.

Reply:
Done.